03_gamedesign/index.tex

```diff
@@ -209,6 +209,20 @@ \subsubsection{Code testing}
 
 While Go is strongly typed and would mean that most errors can be detected at compile-time (or even at time of coding with its performant language server), having unit and integration tests greatly improved the maintainability and ease of development of the project--these issues were particularly important as this was a large-scale group software engineering project--developers stepping on each other's toes is a common occurrence in non-tested group project code. Henceforth, tests were required for non-trivial server-side code.
 
+<<<<<<< HEAD
```

Comment:
this needs to be removed

Reply:
there shouldn't be any edits in gamedesign from team4 as far as I know - is this a mistake?

Reply:
fixing now

Reply:
Done